app/component/CoffeeScoreSlider/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import * as styles from './style.css';
+import clsx from 'clsx';
+import { typography } from 'src/vanilla-extract/typography.css';
+
+interface CoffeeScoreSliderProps {
+  sid: 'flavor' | 'sweetness' | 'acidity' | 'balance' | 'overall';
+}
+
+const TITLE = {
+  flavor: '향미',
+  sweetness: '단맛',
+  acidity: '산미',
+  balance: '밸런스',
+  overall: '종합',
+};
+
+export default function CoffeeScoreSlider({ sid }: CoffeeScoreSliderProps) {
+  const [score, setScore] = useState(5);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = parseInt(event.target.value);
+    setScore(value);
+  };
+
+  return (
+    <section className={styles.scoreDialContainer} aria-label="점수 선택기">
+      <div className={styles.labelContainer}>
+        <label htmlFor={sid} className={typography.heading2}>
+          {TITLE[sid]}
+        </label>
```

Comment:
태그사용을 적절히 잘해주시니, typography 스타일을 사용하는것보다 h2태그를 사용하게 더 적절해보인다 생각했어요.

```suggestion
        <label htmlFor={sid}>
          <h2>{TITLE[sid]}</h2>
        </label>
```

저도 계속 typoegraphy 스타일을 사용했었는데, tag에 맞게 사용하는 버릇을 들여야겠네요

Reply:
오 h2 태그를 사용하면 heading2 스타일이 자동으로 먹히겠네요.
그런데 label 안에 헤더가 들어가는 구조가 적절하지 않은 시맨틱인 것 같은데, 러기는 어떻게 생각하시나요..?